packer/linux/buildkite-ami.json

```diff
@@ -12,7 +12,7 @@
       "region": "{{user `region`}}",
       "source_ami_filter": {
         "filters": {
-          "name": "amzn2-ami-hvm-2.0.*-gp2",
+          "name": "amzn2-ami-hvm-2.0.20220207.1-{{user `arch`}}-gp2",
```

Comment:
Assuming that `2.0.20220207.1` segment is the same for all `arch` that we support (which I'm fairly happy to assume) then that looks good as a temporary solve 👍🏼